src/NuGet.Core/NuGet.Protocol/Utility/OfflineFeedUtility.cs

```diff
@@ -148,115 +148,112 @@ public static void ThrowIfInvalid(string path)
             var source = offlineFeedAddContext.Source;
             var logger = offlineFeedAddContext.Logger;
 
-            using (var packageStream = File.OpenRead(packagePath))
+            using var packageStream = File.OpenRead(packagePath);
+            try
             {
-                try
-                {
-                    var packageReader = new PackageArchiveReader(packageStream, leaveStreamOpen: true);
-                    var packageIdentity = packageReader.GetIdentity();
+                var packageIdentity = default(PackageIdentity);
+                using var packageReader = new PackageArchiveReader(packageStream, leaveStreamOpen: true);
```

Comment:
what's the analyzer fix here?

Reply:
Fix below CA2000 warning. https://github.com/NuGet/NuGet.Client/blob/7d31218575c9ec49aa60bd9a60fa3f6bdad40082/src/NuGet.Core/NuGet.Protocol/GlobalSuppressions.cs#L25